### PR TITLE
Quality: PowerShell Summit GH-3411

### LINKED ISSUE #3411: Quality: pki module formatting fixes part 2
### Prerequisites

- [X] **Existing Issue:**

Search the existing [quality issues][b1] for this repository. If there's an issue that
covers the quality area and articles you want to improve, pick a different quality area
or article set. Don't file a new issue for the same work. Subscribe, react, or comment
on that issue instead.

[b1]: https://github.com/orgs/MicrosoftDocs/projects/15/views/4

- [X] **Descriptive Title:**

Write the title for this issue as a short synopsis. If possible, provide context. For
example, "Quality: Command Syntax in Foo" instead of "Quality: Foo"

### Quality Areas

Formatting code samples, Formatting command syntax

### Article List

- New-CertificateNotificationTask
- New-SelfSignedCertificate
- Remove-CertificateEnrollmentPolicyServer
- Remove-CertificateEnrollmentPolicyServer
- Set-CertificateAutoEnrollmentPolicy
- Switch-Certificate
- Test-Certificate

### PR DESCRIPTION
**Pull request by robderickson**
# PR Summary

This changes fixes formatting in the documentation for the pki module cmdlets listed in issue #3411.

- Fixes #3411 

## PR Checklist

- [x] **Descriptive Title:** This PR's title is a synopsis of the changes it proposes.
- [x] **Summary:** This PR's summary describes the scope and intent of the change.
- [x] **Contributor's Guide:** I have read the [contributors guide][contrib].
- [x] **Style:** This PR adheres to the [style guide][style].

[contrib]: https://learn.microsoft.com/powershell/scripting/community/contributing/overview
[style]: https://learn.microsoft.com/powershell/scripting/community/contributing/powershell-style-guide

### CODE REVIEW

File: docset/winserver2022-ps/pki/New-CertificateNotificationTask.md [hunk 1]
```diff
@@ -11,57 +11,82 @@ title: New-CertificateNotificationTask
...
-Sets the channel of the CertificateServicesClient-Notifications log that will be monitored for certificate lifecycle events.
-The acceptable values for this parameter are:
 
- -- System:  The Operation-System channel will be used.
-This channel should be used to modify system certificate bindings that use computer certificates. 
+Sets the channel of the **CertificateServicesClient-Notifications** log that will be monitored for
+certificate lifecycle events. The acceptable values for this parameter are:
+
+ -- `System`: The **Operation-System** channel will be used. This channel should be used to modify
+ system certificate bindings that use computer certificates.
```
Comment (sdwheeler) [inline docset/winserver2022-ps/pki/New-CertificateNotificationTask.md:83]:
```suggestion
- `System`: The **Operation-System** channel will be used. This channel should be used to modify
  system certificate bindings that use computer certificates.
```

File: docset/winserver2022-ps/pki/New-CertificateNotificationTask.md [hunk 2]
```diff
@@ -11,57 +11,82 @@ title: New-CertificateNotificationTask
...
 
- -- User: The Operational-User channel will be used.
-This channel should be used to modify user certificate bindings.
+ -- `User`: The **Operational-User** channel will be used. This channel should be used to modify
```
Comment (sdwheeler) [inline docset/winserver2022-ps/pki/New-CertificateNotificationTask.md:85]:
```suggestion
- `User`: The **Operational-User** channel will be used. This channel should be used to modify
```

File: docset/winserver2022-ps/pki/New-CertificateNotificationTask.md [hunk 3]
```diff
@@ -11,57 +11,82 @@ title: New-CertificateNotificationTask
 # New-CertificateNotificationTask
 
 ## SYNOPSIS
-Creates a new task in the Task Scheduler that will be triggered when a certificate is replaced, expired, or about to expired.
+
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/pki/New-CertificateNotificationTask.md:14]: The synopsis the only heading we _can't_ have a newline after. It breaks PlatyPS.

```suggestion
```

File: docset/winserver2022-ps/pki/New-CertificateNotificationTask.md [hunk 4]
```diff
@@ -121,16 +149,21 @@ Accept wildcard characters: False
...
+the past. For the notification to be generated both certificates must be present in the store. This
+parameter can only be used with the `Replace` type.
+
+Note: The following warning will be displayed when this this parameter set to `False` and there are
+some certificates in MY store that would have resulted in a notification.
+
+ -- `There are certificates in My store that have been replaced in the past. You can use the
+ New-CertificateNotification cmdlet with the RunTaskForExistingCerts parameter to generate
+ notifications for those certificates to correct any configuration problems that you may already
+ have on this machine.`
```
Comment (sdwheeler) [inline docset/winserver2022-ps/pki/New-CertificateNotificationTask.md:163]:
````suggestion
> [!NOTE]
> The following warning is displayed when this parameter set to `False` and there are some
> certificates in MY store that would have resulted in a notification.
>
> ```Output
> There are certificates in My store that have been replaced in the past. You can use the
> New-CertificateNotification cmdlet with the RunTaskForExistingCerts parameter to generate
> notifications for those certificates to correct any configuration problems that you may already
> have on this machine.
> ```
````

File: docset/winserver2022-ps/pki/New-CertificateNotificationTask.md [hunk 5]
```diff
@@ -11,57 +11,82 @@ title: New-CertificateNotificationTask
...
-Sets the channel of the CertificateServicesClient-Notifications log that will be monitored for certificate lifecycle events.
-The acceptable values for this parameter are:
 
- -- System:  The Operation-System channel will be used.
-This channel should be used to modify system certificate bindings that use computer certificates. 
+Sets the channel of the **CertificateServicesClient-Notifications** log that will be monitored for
+certificate lifecycle events. The acceptable values for this parameter are:
+
+ -- `System`: The **Operation-System** channel will be used. This channel should be used to modify
+ system certificate bindings that use computer certificates.
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/pki/New-CertificateNotificationTask.md:84]:
```suggestion
- `System`: The **Operation-System** channel will be used. This channel should be used to modify
  system certificate bindings that use computer certificates.
```

File: docset/winserver2022-ps/pki/New-CertificateNotificationTask.md [hunk 6]
```diff
@@ -11,57 +11,82 @@ title: New-CertificateNotificationTask
...
 
- -- User: The Operational-User channel will be used.
-This channel should be used to modify user certificate bindings.
+ -- `User`: The **Operational-User** channel will be used. This channel should be used to modify
+ user certificate bindings.
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/pki/New-CertificateNotificationTask.md:86]:
```suggestion
- `User`: The **Operational-User** channel will be used. This channel should be used to modify user
  certificate bindings.
```

File: docset/winserver2022-ps/pki/New-CertificateNotificationTask.md [hunk 7]
```diff
@@ -142,15 +175,18 @@ Accept wildcard characters: False
...
-Specifies the type of events that will trigger certificate notifications.
-The acceptable values for this parameter are:
 
- -- Replace: Certificate replacement events will trigger this notification, including certificates that are renewed by auto-enrollment, using the Certificates snap-in, or by using the Switch-Certificate cmdlet. 
+Specifies the type of events that will trigger certificate notifications. The acceptable values for
+this parameter are:
+
+ -- `Replace`: Certificate replacement events will trigger this notification, including certificates
+ that are renewed by auto-enrollment, using the Certificates snap-in, or by using the
+ `Switch-Certificate` cmdlet.
```
Comment (sdwheeler) [inline docset/winserver2022-ps/pki/New-CertificateNotificationTask.md:184]:
```suggestion
- `Replace`: Certificate replacement events will trigger this notification, including certificates
  that are renewed by auto-enrollment, using the Certificates snap-in, or by using the
  `Switch-Certificate` cmdlet.
```

File: docset/winserver2022-ps/pki/New-CertificateNotificationTask.md [hunk 8]
```diff
@@ -142,15 +175,18 @@ Accept wildcard characters: False
...
 
- -- Expire: Certificate expiration and close-to-expire events will trigger this notification.
+ -- `Expire`: Certificate expiration and close-to-expire events will trigger this notification.
```
Comment (sdwheeler) [inline docset/winserver2022-ps/pki/New-CertificateNotificationTask.md:186]:
```suggestion
- `Expire`: Certificate expiration and close-to-expire events will trigger this notification.
```

File: docset/winserver2022-ps/pki/New-CertificateNotificationTask.md (hunk 4 above)
Comment (michaeltlombardi) [inline docset/winserver2022-ps/pki/New-CertificateNotificationTask.md:163]: @sdwheeler double check me on this one, not sure if I'm right:

```suggestion
> [!NOTE]
> The following warning will be displayed when this this parameter set to `$false` and there are
> some certificates in MY store that would have resulted in a notification.
>
> `There are certificates in My store that have been replaced in the past. You can use the
> New-CertificateNotification cmdlet with the RunTaskForExistingCerts parameter to generate
> notifications for those certificates to correct any configuration problems that you may already
> have on this machine.`
```

File: docset/winserver2022-ps/pki/New-CertificateNotificationTask.md [hunk 9]
```diff
@@ -142,15 +175,18 @@ Accept wildcard characters: False
...
-Specifies the type of events that will trigger certificate notifications.
-The acceptable values for this parameter are:
 
- -- Replace: Certificate replacement events will trigger this notification, including certificates that are renewed by auto-enrollment, using the Certificates snap-in, or by using the Switch-Certificate cmdlet. 
+Specifies the type of events that will trigger certificate notifications. The acceptable values for
+this parameter are:
+
+ -- `Replace`: Certificate replacement events will trigger this notification, including certificates
+ that are renewed by auto-enrollment, using the Certificates snap-in, or by using the
+ `Switch-Certificate` cmdlet.
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/pki/New-CertificateNotificationTask.md:185]:
```suggestion
- `Replace`: Certificate replacement events will trigger this notification, including certificates
  that are renewed by auto-enrollment, using the Certificates snap-in, or by using the
  `Switch-Certificate` cmdlet.
```

File: docset/winserver2022-ps/pki/New-CertificateNotificationTask.md (hunk 8 above)
Comment (michaeltlombardi) [inline docset/winserver2022-ps/pki/New-CertificateNotificationTask.md:186]:
```suggestion
- `Expire`: Certificate expiration and close-to-expire events will trigger this notification.
```

**Review by sdwheeler — CHANGES_REQUESTED** [review]
@robderickson Thanks for the submission. Please see my suggested changes.

File: docset/winserver2022-ps/pki/Remove-CertificateEnrollmentPolicyServer.md [hunk 10]
```diff
@@ -11,52 +11,72 @@ title: Remove-CertificateEnrollmentPolicyServer
...
+The `Remove-CertificateEnrollmentPolicyServer` cmdlet removes an enrollment policy server from the
+current user or local computer configuration. This cmdlet also removes any policy cache file and
+credentials from the vault.
+
+Only one enrollment policy server configuration is removed from the user configured location at a
+time.
 
-If a scope of All is specified and the same URL exists in the local computer (machine) and User contexts, then this cmdlet will fail. 
+If a scope of `All` is specified and the same URL exists in the local computer (machine) and User
+contexts, then this cmdlet will fail.
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/pki/Remove-CertificateEnrollmentPolicyServer.md:35]:
```suggestion
If a scope of `All` is specified and the same URL exists in the local computer (machine) and user
contexts, then this cmdlet will fail.
```

File: docset/winserver2022-ps/pki/Switch-Certificate.md [hunk 11]
```diff
@@ -11,31 +11,42 @@ title: Switch-Certificate
...
+    OldCert = 'Cert:\LocalMachine\My\E42DBC3B3F2771990A9B3E35D0C3C422779DACD7'
+    NewCert = 'Cert:\LocalMachine\My\4A346B4385F139CA843912D358D765AB8DEE9FD4'
+}
+Switch-Certificate @params
 ```
 
-This example sets the renewal property of the certificate with the thumbprint E42DBC3B3F2771990A9B3E35D0C3C422779DACD7 as renewed by the certificate with the thumbprint 4A346B4385F139CA843912D358D765AB8DEE9FD4 and generates a replace certificate notification.
+This example sets the renewal property of the certificate with the thumbprint
+E42DBC3B3F2771990A9B3E35D0C3C422779DACD7 as renewed by the certificate with the thumbprint
+4A346B4385F139CA843912D358D765AB8DEE9FD4 and generates a replace certificate notification.
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/pki/Switch-Certificate.md:44]:
```suggestion
This example sets the renewal property of the certificate with the thumbprint
`E42DBC3B3F2771990A9B3E35D0C3C422779DACD7` as renewed by the certificate with the thumbprint
`4A346B4385F139CA843912D358D765AB8DEE9FD4` and generates a replace certificate notification.
```

File: docset/winserver2022-ps/pki/Switch-Certificate.md [hunk 12]
```diff
@@ -11,31 +11,42 @@ title: Switch-Certificate
...
 
 ### EXAMPLE 2
-```
-PS C:\>Set-Location -Path cert:\LocalMachine\My
+
+```powershell
+Set-Location -Path cert:\LocalMachine\My
 
 PS cert:\LocalMachine\My>$oldCert = Get-ChildItem -Path E42DBC3B3F2771990A9B3E35D0C3C422779DACD7
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/pki/Switch-Certificate.md:51]: I can't comment on the line after this one, but we should remove the prompt from all lines in this example and remove the extra newlines.

File: docset/winserver2022-ps/pki/Test-Certificate.md [hunk 13]
```diff
@@ -120,12 +139,13 @@ Accept wildcard characters: False
 ```
 
 ### -Policy
-Specifies the policies that will be applied to verify the certificate.
-The acceptable values for this parameter are: AUTHENTICODE, BASE, NTAUTH, and SSL.
-If this parameter is not specified, then the BASE policy is used.
+
+Specifies the policies that will be applied to verify the certificate. The acceptable values for
+this parameter are: `AUTHENTICODE`, `BASE`, `NTAUTH`, and `SSL`. If this parameter is not specified,
+then the BASE policy is used.
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/pki/Test-Certificate.md:145]:
```suggestion
then the `BASE` policy is used.
```

**Review by michaeltlombardi — CHANGES_REQUESTED** [review]

File: docset/winserver2022-ps/pki/New-CertificateNotificationTask.md (hunk 3 above)
Comment (robderickson) [reply docset/winserver2022-ps/pki/New-CertificateNotificationTask.md]: 🤦‍♂️ I'll update all articles and re-push. Thanks!

File: docset/winserver2022-ps/pki/Switch-Certificate.md (hunk 12 above)
Comment (robderickson) [reply docset/winserver2022-ps/pki/Switch-Certificate.md:51]: Whoops. I misread this: https://learn.microsoft.com/en-us/powershell/scripting/community/contributing/powershell-style-guide?view=powershell-7.3#avoid-using-powershell-prompts-in-examples

I'll fix these along with the Synopsis fix..

**Comment by robderickson** [comment]
@michaeltlombardi I think your comments can be resolved now. A lot of them were the same as Sean's, but I left them unresolved until you have a chance to re-review. Thanks!

**Review by michaeltlombardi — APPROVED** [review]

**Pull request merged**